util.go

```diff
@@ -102,3 +104,61 @@ func sortByteSlices(src [][]byte) [][]byte {
 	sort.Sort(bzz)
 	return bzz
 }
+
+// Colors: ------------------------------------------------
+
+const (
+	ANSIReset  = "\x1b[0m"
+	ANSIBright = "\x1b[1m"
+
+	ANSIFgGreen = "\x1b[32m"
+	ANSIFgBlue  = "\x1b[34m"
+	ANSIFgCyan  = "\x1b[36m"
+)
+
+// color the string s with color 'color'
+// unless s is already colored
+func treat(s string, color string) string {
+	if len(s) > 2 && s[:2] == "\x1b[" {
+		return s
+	}
+	return color + s + ANSIReset
+}
+
+func treatAll(color string, args ...interface{}) string {
+	parts := make([]string, 0, len(args))
+	for _, arg := range args {
+		parts = append(parts, treat(fmt.Sprintf("%v", arg), color))
+	}
+	return strings.Join(parts, "")
+}
+
+func Green(args ...interface{}) string {
+	return treatAll(ANSIFgGreen, args...)
+}
+
+func Blue(args ...interface{}) string {
+	return treatAll(ANSIFgBlue, args...)
+}
+
+func Cyan(args ...interface{}) string {
+	return treatAll(ANSIFgCyan, args...)
+}
+
+func ColoredBytes(data []byte, textColor, bytesColor func(...interface{}) string) string {
+	colors := os.Getenv("COLORS_ON")
```

Comment:
Mmmmh... If we really want go for an environment variable let's at least prefix it with `IAVL_`:

```suggestion
	colors := os.Getenv("IAVL_COLORS_ON")
```